### ISSUE TITLE
Optional close unused controls in background

### ISSUE THREAD

**Issue by vladimirlapikov** [florent37/Flutter-AssetsAudioPlayer#114]
Hi I have no use for prev forward buttons  in background player ,can you add optional feature to choose what controls I want in background?
thnx

**Comment by florent37** [comment]
Yes this was in my to-do list, but thanks to added this into issue, I will not forget it ;)

**Comment by vladimirlapikov** [comment]
Thank you for your work,this is best plugin for flutter for audio,just for you to know :)

**Comment by florent37** [comment]
published in `1.6.2+1`


```dart
open(AUDIO,
   showNotification: true,
   notificationSettings: NotificationSettings(
       prevEnabled: false, //disable the previous button
  
       //and have a custom next action (will disable the default action)
       customNextAction: (player) {
         print("next");
       }
   )

)
```

**Comment by florent37** [comment]
thanks for the message btw :)

**Comment by florent37** [comment]
did this "notificationSettings" solve your problem ? :)

**Comment by florent37** [comment]
if it does not work it's maybe a flutter cache error, can you try this : https://github.com/flutter/flutter/issues/52739#issuecomment-600566849

**Comment by vladimirlapikov** [comment]
Hi thanks for quick solving, I'm checking right now

**Comment by vladimirlapikov** [comment]
Hi , works very well, there is one moment - > in the "NotificationSettings" when I want to disable more than one background button-> whole background notification service is falling.
Let's say I want in my case close "prev", "next" and "stop" buttons and I cannot

**Comment by florent37** [comment]
Ok I'll try again with only playPause enable

**Comment by vladimirlapikov** [comment]
Thank you :)))

**Comment by florent37** [comment]
I fixed it in 1.6.2+3 :)
